### PR TITLE
Update DG: Add READ features use cases

### PR DESCRIPTION
**Pull request by ruo-x**
Use cases added is listed below:
1. List all volunteering events (UC02)
2. Read an individual event (UC03)

use cases are numbered anyhow though lol

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -295,28 +295,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 
 **Extensions**
 
+* 1a. The input command is incorrect.
+
+    * 1a1. System prompts the user to provide a valid command.
+      
+      Use case resumes from step 1.
+
 * 2a. The list is empty.
```
Comment (zacwong2151) [inline docs/DeveloperGuide.md:317]: Correct me if I'm wrong but this should be `1b`

**Review by zacwong2151 — CHANGES_REQUESTED** [review]
LGTM apart from one issue

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (brein62) [reply docs/DeveloperGuide.md:317]: `1a` is fine. this is the first extension for 1

**Review by brein62 — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zacwong2151) [reply docs/DeveloperGuide.md:317]: I meant that `2a` shd be `1b` instead cos `The list is empty` happens after step 1, not after step 2. Thoughts?

**Review by zacwong2151 — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (brein62) [reply docs/DeveloperGuide.md:317]: hmm in this case i agree, though i would put the extension as `There are no events to list`

**Review by brein62 — CHANGES_REQUESTED** [review]
can change 2a to 1b per @zacwong2151

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (zacwong2151) [reply docs/DeveloperGuide.md:317]: yup makes sense @ruo-x

Comment (jordankanghm) [reply docs/DeveloperGuide.md:317]: agreee

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -295,28 +295,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `iVolunteer` and the **Actor** is the `Volunteer Coordinator`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case 2: List all volunteering events**
```
Comment (jordankanghm) [inline docs/DeveloperGuide.md:300]: I think can rename `Use Case 2` to `Use Case UCE02 to align with our formats`

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -295,28 +295,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+    * 1a1. System prompts the user to provide a valid command.
+      
+      Use case resumes from step 1.
+
 * 2a. The list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+**Use case 3: Read an individual event**
```
Comment (jordankanghm) [inline docs/DeveloperGuide.md:321]: Same formatting here, `Use Case UCE03`

**Review by jordankanghm — CHANGES_REQUESTED** [review]
Just some minor issues!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ruo-x) [reply docs/DeveloperGuide.md:317]: i dunno i followed the AB3 original one and they put it at 2a tho

Comment (ruo-x) [reply docs/DeveloperGuide.md:317]: coz 1b makes sense too ill j change it

**Review by brein62 — APPROVED** [review]
LGTM!

**Review by tiongMax — APPROVED** [review]
lgtm!

**Comment by jordankanghm** [comment]
lotm

**Review by jordankanghm — APPROVED** [review]
lotm

**Pull request merged**